src/uct/ib/rc/verbs/rc_verbs_common.c

```diff
@@ -23,12 +23,10 @@ ucs_config_field_t uct_rc_verbs_iface_common_config_table[] = {
    "a minimum between this value and the TX queue length. -1 means no limit.",
    ucs_offsetof(uct_rc_verbs_iface_common_config_t, tx_max_wr), UCS_CONFIG_TYPE_UINT},
 
-#if IBV_EXP_HW_TM
   {"TM_SYNC_RATIO", "0.5",
    "Maximal portion of the tag matching list which can be canceled without requesting\n"
    "a completion.",
-   ucs_offsetof(uct_rc_verbs_iface_common_config_t, tm.sync_ratio), UCS_CONFIG_TYPE_DOUBLE},
-#endif
+   ucs_offsetof(uct_rc_verbs_iface_common_config_t, tm_sync_ratio), UCS_CONFIG_TYPE_DOUBLE},
```

Comment:
Why do we need this env var if  no HW TM code?

Reply:
prefer to avoid too many #ifdef in the code, if no env var will need to remove also iface->tm_sync_ratio and wrap all code which uses it with #ifdef

Reply:
will it be to many places? Other TM variables are guarded with `#if IBV_EXP_HW_TM` (in rc_iface.c)

Reply:
i think that eventually need to guard only what is using ibv_exp_ functions so this PR is a step in that direction

Reply:
fixed